src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/require-default-props */
+import React, { useState, useEffect } from "react";
+import ConfirmationPopup, { TermsOfUse, Button } from "shared-components";
+import { getContent } from "../../../modules/csoAccountAgreementContent";
+import { propTypes } from "../../../types/propTypes";
+
+const continueButton = {
+  label: "Create CSO Account",
+  styling: "normal-blue btn",
+  onClick: () => {}
+};
+
+const content = getContent();
+
+export default function CSOAccount({ confirmationPopup }) {
+  const [termsAccepted, acceptTerms] = useState(false);
+  const [continueBtnEnabled, setContinueBtnEnabled] = useState(false);
+
+  useEffect(() => {
+    if (termsAccepted) {
+      setContinueBtnEnabled(true);
+    } else {
+      setContinueBtnEnabled(false);
+    }
+  }, [termsAccepted]);
+
+  return (
+    <>
```

Comment:
empty?

Reply:
yeah its just a fragment :) so empty is good! since we cant have adjacent jsx elements without being contained in something